wls-gui-wahllokalsystem/tests/composables/infomanagement/konfigurationsparameterMapper.spec.ts

```diff
@@ -20,9 +20,11 @@ describe("konfigurationsparameterMapper.ts", () => {
       const dto: KonfigurationDTO[] = createKonfigurationDtoList(3);
 
       const expectedModel: Konfigurationsparameter[] = [
-        { schluessel: dto[0].schluessel, wert: dto[0].wert },
-        { schluessel: dto[1].schluessel, wert: dto[1].wert },
-        { schluessel: dto[2].schluessel, wert: dto[2].wert },
+        /* eslint-disable @typescript-eslint/no-non-null-assertion -- Temporarily disabling because only valid dto is given as input */
```

Comment:
`Temporarily disabling ` vermutlich mit Bezug auf den Test, ich war etwas verwirrt warum temporär, wann wird es wieder aktiviert bzw. was ist zu tun
```suggestion
        /* eslint-disable @typescript-eslint/no-non-null-assertion -- disabling because only valid dto is given as input */
```